spec/feature/items_view_spec.rb

```diff
@@ -30,8 +41,23 @@
     expect(current_path).to eq(items_path)
   end
 
+  it "can update the item" do
+    visit(item_path(item))
+    click_link_or_button("Edit Item")
+    fill_in('item[title]', with: "Racoon Ragu" )
```

Comment:
Space inside parentheses detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.